students/student_framllat/session02/fizzbuzz.py

```diff
@@ -0,0 +1,15 @@
+import random
+
+for i in range(1,10):
+
+    test_num = random.randint(1,100)
```

Comment:
interesting -- why randomize? I think the problem description simple says the numbers from one to 100